pkg/bootstrapper/bootstrapper.go

```
@@ -667,6 +667,19 @@ func (wmcb *winNodeBootstrapper) Disconnect() error {
 	return err
 }
 
+// UninstallKubelet uninstalls the kubelet service from Windows node
+func (wmcb *winNodeBootstrapper) UninstallKubelet() error {
+	if wmcb.kubeletSVC == nil {
+		return fmt.Errorf("kubelet service is not present")
```

We should remember to ignore this error when using this in WMCO.